README.md

```diff
@@ -160,13 +160,18 @@ We are using Dalmatian to send request to Terra,
 
 #### CN
 
-We are running a set of 5 functions/workflows To generate the copy number dataset:
-
-*   **BamToUnmappedRGBams_MC** vdauwera/BamToUnmappedRGBamsSnapshot ID: 3
-*   **Generate_uBAM_File_List** gkugener/ArrayOfFilesToTxtSnapshot ID: 1
-*   **Realign_WES_GATK4** gatk/PreProcessingForVariantDiscovery_GATK4Snapshot ID: 7
-*   **CNV_sample_XX** gatk/CNV_Somatic_Pair_WorkflowSnapshot ID: 9
-*   **Aggregate_CN_seg_files** gkugener/Aggregate_CN_seg_filesSnapshot ID: 2
+We are running a set of 5 functions/workflows To generate the copy number dataset in the following order:
+
+1. **BamToUnmappedRGBams_MC** vdauwera/BamToUnmappedRGBamsSnapshot ID: 3
+2. **Generate_uBAM_File_List** gkugener/ArrayOfFilesToTxtSnapshot ID: 1
+3. **Realign_WES_GATK4** gatk/PreProcessingForVariantDiscovery_GATK4Snapshot ID: 7
+4. **CNV_sample_XX** [gatk/CNV_Somatic_Pair_WorkflowSnapshot ID: 9](https://portal.firecloud.org/?return=terra#methods/gatk/CNV_Somatic_Pair_Workflow/9/wdl): This pipeline recieves read counts, which are the output of previous step, and determines the copy number segments and assigns them the deletion/amplification/neutral status. The pipeline requires an input PoN, which currently is defaulted on [hg38 ICE WES capture kit](gs://ccle_default_params/pons/hg38_ice_pon_XX.pon.hdf5). The following tools are run in sequence:
+   1. [*gatk DenoiseReadCounts*](https://gatk.broadinstitute.org/hc/en-us/articles/360037593691-DenoiseReadCounts): Denoises read counts to produce denoised copy ratios
+   2. [*gatk ModelSegments*](https://gatk.broadinstitute.org/hc/en-us/articles/360036350172-ModelSegments): Models segmented copy ratios from denoised read counts and segmented minor-allele fractions from allelic counts
+   3. [*gatk CallCopyRatioSegments*](https://gatk.broadinstitute.org/hc/en-us/articles/360036730311-CallCopyRatioSegments): Calls copy-ratio segments as amplified, deleted, or copy-number neutral
+   4. *PlotDenoisedCopyRatios*
+   5. *PlotModeledSegments*
```

Comment:
Are we using these or should they be removed?

Reply:
They are part of the pipeline but we do not provide them in the portal for now. I think we should still add them. 
Additionally, I think it might be better to provide just a link to the workflow which is itself public and documented. And do the same thing for all the workflows we use.

Reply:
Do you mean pages like this?
https://portal.firecloud.org/?return=terra#methods/gatk/CNV_Somatic_Pair_Workflow/9